src/converters.js

```diff
@@ -779,7 +779,7 @@ export function modelIndentPasteFixer( evt, [ content, selection ] ) {
 			if ( indentChange > 0 ) {
 				// Adjust indent of all "first" list items in inserted data.
 				while ( item && item.is( 'listItem' ) ) {
-					item.setAttribute( 'indent', item.getAttribute( 'indent' ) + indentChange );
+					item._setAttribute( 'indent', item.getAttribute( 'indent' ) + indentChange );
```

Comment:
This is done in a callback to `model#insertContent`. Should we use model `change` block here? cc @pjasiun

Reply:
I created a follow-up about it https://github.com/ckeditor/ckeditor5-engine/issues/1332.